changelog.d/20834_nats_jetstream_sink.feature.md

```diff
@@ -0,0 +1,18 @@
+Add possibility to use NATS JetStream in NATS sink. Can be turned on/off via `jetstream` option (default is false).
+
+### Example
+
+#### Config
+
+```
+sinks:
+  nats:
+    type: nats
+    inputs:
+      - in
+    subject: nork
+    url: "nats://localhost:4222"
+    jetstream: true
+    encoding:
+      codec: json
+```
```

Comment:
The changelog should be fairly succinct, so we don't need the example here. We do need the authors included for external contributors.

```suggestion
authors: whatcouldbepizza
```

Reply:
Also can you make sure it ends with a newline please.